glyphs2fontir/src/glyphdata.rs

```diff
@@ -4,8 +4,705 @@
 
 use std::collections::HashSet;
 
-pub(crate) fn is_nonspacing_mark(codepoints: &HashSet<u32>, name: &str) -> bool {
-    // Try first by name
+#[inline(always)]
+pub(crate) fn might_be_a_nonspacing_mark_name(name: &str) -> bool {
```

Comment:
Can we capture the rationale and the process here, as a comment?

Reply:
Added a terse comment. We also refer the reader to the generating python at the top.